Cargo.toml

```diff
@@ -24,21 +24,25 @@ regex = "~1.3"
 env_logger = "~0.7"
 num-traits = "~0.2"
 num-derive = "~0.3"
-diesel = { version = "~1.4", features = ["postgres", "uuid", "numeric", "chrono"] }
+diesel = { version = "~1.4", features = ["uuid", "numeric", "chrono"] } # target-specific features are coming, in the meantime keep postgres disabled to compile on android (https://github.com/rust-lang/cargo/issues/7914)
```

Comment:
Can we pls add postgres as a target-specific feature right away?

Reply:
i would just wait a little bit more, because there's no way to say in the Cargo.toml that the project should be compiled with a specific cargo command line. and if somebody forgets to add the required flags to cargo, it will just behave differently without giving any warnings or errors.

it's not like using a nightly compiler which is very explicit and you get an error if you forget to do it. it's more subtle and confusing imho